app/models/server/models/ServerEvents.ts

```diff
@@ -0,0 +1,10 @@
+import { Base } from './_Base';
+
+export class ServerEvents extends Base {
+	constructor() {
+		super('server_events');
+		this.tryEnsureIndex({ ip: 1, ts: 1 });
```

Comment:
Are we saving all the login attempts forever?

Reply:
@rodrigok yes